clients/drcachesim/tracer/syscall_pt_trace.h

```diff
@@ -198,6 +198,16 @@ class syscall_pt_trace_t {
      * every syscall in the current thread.
      */
     file_t output_file_;
+
+    /* The ring buffer that stores the recorded PT data is assigned a size of
+     * (1 << ring_buffer_size_shift_) * page_size.
+     * For apps with a high thread count, this may cause us to exceed the available
+     * memory. But if it is configured too low, we may see errors of type
+     * READ_RING_BUFFER_ERROR_OLD_DATA_OVERWRITTEN or
+     * "get next instruction error: expected tracing enabled event" during PT trace
+     * decoding.
```

Comment:
Are these propagated to the user?  Will the user know what they mean?  Please augment the option docs to explain what error to look for to identify when the buffer is too small.  Also augment the error message returned to add a user-friendly message suggesting to increase the new option.

Reply:
Done.